charm/ksql/config.yaml

```diff
@@ -7,15 +7,11 @@ options:
     type: string
     default: development
     description: The KSQL cluster id that the KSQL server will be assigned to.
-  ksql-listener-port:
-    type: int
-    default: 8088
-    description: The port where the REST server will listen on
-  ksql-listener-protocol:
-    type: string
-    default: http
-    description: The protocol that the REST server will listenn on
   use-ssl:
```

Comment:
This option should be removed. The charm does not seem to implement setting up an HTTPS listener. That would need to be done with certificates and keys obtains from easyrsa.

Reply:
It's still there as far as I can tell.